app/src/main/java/com/example/puzzlepic/dto/Picture.kt

```diff
@@ -5,13 +5,19 @@ import com.google.gson.annotations.SerializedName
 import kotlinx.android.parcel.Parcelize
 
 
-data class Picture(@SerializedName("id") var pictureID : String = "", @SerializedName("alt_description") var pictureTitle : String, @SerializedName("created_at") var date : String, var location : String, var notes : String,
-                        var puzzleID : Int = 0, @SerializedName("urls") var urls : UnsplashPhotoUrls ){
-        override fun toString() : String {
-            return pictureTitle
-            return date
-        }
+data class Picture(
+    @SerializedName("id") var pictureID : String = "",
+    @SerializedName("alt_description") var pictureTitle : String,
+    @SerializedName("created_at") var date : String,
+    var location : String, var notes : String,
+    var puzzleID : Int = 0,
+    @SerializedName("urls") var urls : UnsplashPhotoUrls
+) {
+    override fun toString() : String {
+        return pictureTitle
+        return date
```

Comment:
There should only be one return statement.  Did you mean to make this:
return "$pictureTitle $date" 
instead?